frontend/guide/POLICIES.md

```diff
@@ -0,0 +1,92 @@
+5. Enter the list of Row Access Tags. Row access tags are simple string filters that are used by tables/views associated with the policy in order to filter rows for users*.
+6. Click 'SAVE'. Details on the 'Marketplace' section will be covered in the [Integrating with Marketplace](#integrating_with_marketplace) section.
+
+\* Row Access Tags currently only work for users, they do not work with groups.
+
+## <a name="editing_a_policy">Editing a Policy</a>
+1. Click on the pencil icon in the rightmost column.
+2. Make the necessary modifications. Ensure caution as an edit to an existing policy will impact all users entitled to it.
+4. Click 'Save'.
+
+## <a name="deleting_a_policy">Deleting a Policy</a>
+Ensure caution when deleting any policy. If you have GCP Marketplace integration set up, this can break any solution or user associations for purchased solutions that you are selling through marketplace.
+
+1. Click the trash image on the rightmost side.
+2. Click 'DELETE' to confirm the deletion.
+
+## <a name="integrating_with_marketplace">Integrating with Marketplace</a>
+In order to integrate Datashare with the GCP Marketplace, you must first sign up to become a [Google Cloud Marketplace vendor](https://cloud.google.com/marketplace/docs/partners). Once you have been approved as a partner you may then list your data solutions within Marketplace.
+
+In order to create a data solution in GCP Marketplace and sell it, follow the instructions below:
+
+1. Go to [Partner Portal](https://console.cloud.google.com/partner/solutions).
+2. On the top navigation menu, cick '+ ADD SOLUTION'.
+3. Enter a solution name, select solution type of 'Managed service' and pricing type 'Billed by Google'.
+4. Go to edit for 'SOLUTION METADATA' and fill in the required information and click 'Save'.
+    * If you do not yet want to make your solution public in the GCP Marketplace, ensure the 'Hide solution from end users' checkbox is checked.
+5. Go to edit for 'SOLUTION DETAILS' and fill in the required information and click 'Save'.
+6. Go to edit for 'PLANS & FEATURES'. For more information on this section see [Plans & Features](#plans_and_features).
+    * For the Signup URL, use the following: ```https://[YOUR_DATASHARE_DOMAIN]/v1alpha/projects/[YOUR_GCP_PROJECT]/accounts:register```
+    * Leave the Account Recovery URL empty.
+    * Leave SSO Login disabled.
+    * For the Dashboard URL, use the following: ```https://[YOUR_DATASHARE_DOMAIN]/myDashboard```
+    * Add your [service features](#service_features). Service features can be included in multiple plans.
+    * Configure your [service plans](#service_plans). Service plans can encompass multiple features.
+7. Complete the billing on-boarding spreadsheet here - TBD. The billing setup process will take up to two weeks to complete. The GCP Marketplace team will let you know once billing setup has been completed and you can proceed with the next steps.
```

Comment:
Can we change this to something along the lines of *Reach out to the GCP Marketing partner team vi <EMAIL-ADDRESS>*?

Reply:
Updated - this section needs more detail. I'll work with the marketplace team to get the necessary details that we need to add here relating to billing.